contrib/completion/bash/docker

```diff
@@ -1155,7 +1155,20 @@ __docker_complete_plugin() {
 			resultArray+=( "$value" )
 		fi
 	done
-	local result=$(eval "${resultArray[*]}" 2> /dev/null | grep -v '^:[0-9]*$')
+	local rawResult=$(eval "${resultArray[*]}" 2> /dev/null)
+	local result=$(grep -v '^:[0-9]*$' <<< "$rawResult")
+
+	# Compose V2 completions sometimes returns returns `:8` (ShellCompDirectiveFilterFileExt)
+	# with the expected file extensions (such as `yml`, `yaml`) to indicate that the shell should
+	# provide autocompletions for files with matching extensions
+	local completionFlag=$(echo "${rawResult[@]}" | tail -1)
```

Comment:
Since `rawResult` is just a string, this can be a little simpler (and avoid some of `echo`'s potentially pathologic behavior):
```suggestion
	local completionFlag=$(tail -1 <<< "$rawResult")
```

(Is it _always_ the last entry?  I guess we'd never actually have a file extension like `:0-9`)

Reply:
@thaJeztah I was just going to take 5 mins to try the changes @tianon suggested before amending, but since I'm out I'm okay if you want to do that/commit the changes and merge yourself

Reply:
No worries, I'll have a look! Let's wait for that before merging (assuming @tianon's changes look good to everyone) /cc @albers